tempesta_fw/http.c

```diff
@@ -2712,6 +2743,14 @@ tfw_http_req_process(TfwConn *conn, const TfwFsmData *data)
 			       && (req->content_length != req->body.len));
 		}
 
+		/* Assign the right Vhost for this request. */
+		if (tfw_http_req_set_context(req)) {
```

Comment:
So before vhost is assigned to the request, frang configuration from default vhost is used. Looks like default vhost frang settings must be less strict than non-default vhost frang.  This should be documented in wiki.